### ISSUE TITLE
[Rule Guideline] Rule 8 is not worded well and refers to email being sent

### ISSUE THREAD

**Issue by lcn2** [ioccc-src/winner#37]
### Is there an existing issue for this?

- [X] I have searched for existing issues and did not find anything like it

### Which Rule and/or Guideline has a problem?

[Rule 8](https://www.ioccc.org/next/rules.html#rule8):

```
Entries must be received prior to close of the contest is [open](https://www.ioccc.org/status.html#open).

This contest is scheduled to open on **2025-01-31 23:19:17.130705 UTC**
and close on **2025-04-01 23:29:31.374143 UTC**.

A confirmation of submission will be sent to the submitting email address
before the close of the contest.
```

### What is wrong with the Rule and/or Guideline?

The wording is awkward.  It also implies that the email will be sent.  It also uses the term "Entries" when it should refer to "submission".

### How do you recommend we fix the problem?

Use this text for Rule 8:

```
Submissions may only be uploaded to the [IOCCC submit server](https://submit.ioccc.org) while the contest is **[open](../status.html#open)**.

Once the contest is in the **[judging](../status.html#judging)** state (or **[closed](../status.html#closed)**, you may **NOT** upload submissions.

See the [Important IOCCC dates](#dates) above for when these contest states are scheduled.
```

### Anything else?

fizzbin ?

**Comment by lcn2** [comment]
This issue has been resolved with commit c5d87e9f674879dd5d87f8dcd5eac98df8c1779f

**Comment by xexyl** [comment]
Thanks!
